test/integration/serverless_perf/compute.sh

```diff
@@ -12,16 +30,17 @@ do
     dockerId=$(docker run -d datadogci/lambda-extension)
     sleep 10
     numberOfMillisecs=$(docker logs "$dockerId" | grep 'ready in' | grep -Eo '[0-9]{1,4}' | tail -3 | head -1)
-    totalMs=$((totalMs+numberOfMillisecs))
-    echo "Iteration $i - Statup time = $numberOfMillisecs"
+    startupTimes+=($numberOfMillisecs)
+    log "Iteration=$i | Startup Time=$numberOfMillisecs"
 done
 
-echo "Total computed : $totalMs"
-echo "Threshold : $TOTAL_THREESHOLD"
+medianMs=$(calculate_median "${startupTimes[@]}")
+
+log "Median=$medianMs | Threshold=$STARTUP_TIME_THRESHOLD"
 
-# check whether or not the total duration exceeds the threshold
-if (( totalMs > TOTAL_THREESHOLD )); then
+# check whether or not the median duration exceeds the threshold
+if (( medianMs > STARTUP_TIME_THRESHOLD )); then
     exit 1
```

Comment:
Can we include a log message here before exiting? Something like "median value is above threshold"? And maybe print out all of the values as well? I want to ensure that someone new to our team can easily understand what went wrong and hopefully how to address it.

Reply:
I added a log message on failure. There are already additional logs printed on each iteration that should meet this use case